catalog/app/embed/File.js

```diff
@@ -429,12 +428,20 @@ export default function File({
         callback(AsyncResult.Err(Preview.PreviewError.InvalidVersion({ handle }))),
     })
 
+  const scoped = ecfg.scope && path.startsWith(ecfg.scope)
+  const scopedPath = scoped ? path.substring(ecfg.scope.length) : path
+  const crumbs = BreadCrumbs.use(
+    s3paths.up(scopedPath),
+    (segPath) => urls.bucketDir(bucket, `${scoped ? ecfg.scope : ''}${segPath}`),
```

Comment:
this function is used as a memo arg -- memoization won't work unless you memoize this function as well